apps/src/applab/applab.js

```diff
@@ -147,7 +147,10 @@ function loadLevel() {
   // This led to lots of hackery in the code to properly scale the visualization
```

Comment:
Do you think adding widget mode will re-introduce any problems like this?

Reply:
Normally, yes, but we dodge that bullet here by not allowing resizing. The hackery that this comment is referring to is a series of work-arounds that allow dynamic resizing. For widget mode, we use fixed widths of 600px.

More details are captured in a slack conversation starting here: https://codedotorg.slack.com/archives/C0T0UQPLZ/p1564511130033400